src/test/java/pageObjects/CreateDevicePageTests.java

```diff
@@ -0,0 +1,114 @@
+package pageObjects;
+
+import com.sun.source.tree.AssertTree;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class CreateDevicePageTests extends BaseTest {
+
+
+    @Test
+    public void successfulCreateDevice() {
+        createDevicePage.successfullyCreateDevice();
+        WebElement successToaster = driver.findElement(By.id("success_toaster"));
+        Assert.assertEquals(successToaster.getText(), "Item created successfully!");
```

Comment:
if we successfully create a device, could we also check if this device was created and is shown in the table?